packages/core/types/src/order/service.ts

```diff
@@ -1512,6 +1515,13 @@ export interface IOrderModuleService extends IModuleService {
     sharedContext?: Context
   ): Promise<void>
 
+  /*
+  cancelReturn(
+    returnData: CancelOrderReturnDTO,
+    sharedContext?: Context
+  ): Promise<void>
+  */
```

Comment:
question: Coming later or safe to remove?

Reply:
This is coming later. The return will have its own id, and this is not yet implemented in the module.

Reply:
👍